scripts/build_antithesis_images.sh

```diff
@@ -47,7 +47,7 @@ source "${AVALANCHEGO_CLONE_PATH}"/scripts/lib_build_antithesis_images.sh
 build_antithesis_builder_image "${GO_VERSION}" "antithesis-subnet-evm-builder:${IMAGE_TAG}" "${AVALANCHEGO_CLONE_PATH}" "${SUBNET_EVM_PATH}"
 
 # Ensure avalanchego and subnet-evm binaries are available to create an initial db state that includes subnets.
-"${AVALANCHEGO_CLONE_PATH}"/scripts/build.sh
+pushd "${AVALANCHEGO_CLONE_PATH}" && ./scripts/build.sh && popd
```

Comment:
What does this do? Why is this change needed (asking out of curiosity really)

Reply:
Yeah... this was another 20 minute adventure.
See this commit: https://github.com/ava-labs/subnet-evm/pull/1401/commits/33bd19399ea863babec7b7e4b6e4a57e599afb20

For some reason, (perhaps since we dropped the circular import and import less stuff), go.sum for subnet-evm doesn't contain "github.com/jackpal/gateway" anymore.
Therefore if we trigger the avalanchego build script from subnet-evm's repo root (it gets checked out to avalanchego/) it doesn't work (I think these scripts assume they are run from the repo root).

Reply:
What on earth! I would prefer funny shell scripting than having a blank import for `github.com/jackpal/gateway` really, if that's possible. Weird adventure yet again!